app/src/main/java/io/intrepid/contest/screens/conteststatus/ContestStatusPresenter.java

```diff
@@ -7,15 +7,17 @@
 import io.intrepid.contest.R;
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.rest.ContestResponse;
 import io.intrepid.contest.rest.ContestStatusResponse;
 import io.reactivex.Observable;
 import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
 import timber.log.Timber;
 
 class ContestStatusPresenter extends BasePresenter<ContestStatusContract.View> implements ContestStatusContract.Presenter {
 
-    private static final int API_CALL_INITIAL_DELAY = 0;
-    private static final int API_CALL_INTERVAL = 2;
+    private static final long API_CALL_INITIAL_DELAY = 0;
```

Comment:
I think it was a suggestion when I was making tests, but it was unnecessary! I'll change it back.